using-kubecost/navigating-the-kubecost-ui/external-costs.md

```diff
@@ -0,0 +1,90 @@
+# External Costs
+
+{% hint style="info" %}
+External Costs is currently in beta.
+{% endhint %}
+
+External Costs is a monitoring dashboard for third party service costs that are not directly from cloud providers. Currently, this includes monitoring for Datadog costs. More third party services are expected to be supported in the future.
+
+![External Costs](/images/external-costs.png)
+
+## Enabling External Costs
+
+Kubecost will require the integration of a service's plugin in order to detect and display costs associated with that service.
+
+From your Datadog account, you will need the following values:
+
+* `datadog_site`: ex. us5.datadoghq.com
+* `datadog_api_key`: ex. c508d4fd3d126abbbbdc2fe96b0f6613
+* `datadog_app_key`: ex. f357b1f4efefb0870109e0d1aa0cb437b5f10ab9
+
+See Datadog's [API and Application Keys](https://docs.datadoghq.com/account_management/api-app-keys/) for help finding your API and application key values.
+
+Now append your *values.yaml* file with the following, replacing the example Datadog values with your own. It is also crucial the following Helm flags are set to `true`:
+
+* `kubecostModel.plugins.enabled`
+* `kubecostModel.plugins.install.enabled`
+
+```
+plugins:
+    enabled: true
+    install:
+      enabled: true
+      fullImageName: curlimages/curl:latest
+      securityContext:
+        allowPrivilegeEscalation: false
+        seccompProfile:
+          type: RuntimeDefault
+        capabilities:
+          drop:
+          - ALL
+        readOnlyRootFilesystem: true
+        runAsNonRoot: true
+        runAsUser: 1000
+    folder: /opt/opencost/plugin
+    # leave this commented to always download most recent version of plugins
+    # version: <INSERT_SPECIFIC_PLUGINS_VERSION>

```

Comment:
Is there a default version we can use, or does the user still need to visit opencost-plugin and find the most recent version?

Reply:
There's no default, and there's no plan for doing any kind of LTS for any plugin versions.

Generally, users should keep this commented out though. They'll almost always need the latest version.